packages/api-db-mongodb/src/migration.ts

```diff
@@ -789,6 +797,73 @@ export const Migrations: Migration[] = [
       await images.updateMany({}, {$rename: {source: 'link'}})
       await images.updateMany({}, {$rename: {author: 'source'}})
     }
+  },
+  {
+    // change embed block properties width and height from number to string
+    version: 20,
+    async migrate(db) {
+      const articles = db.collection(CollectionName.Articles)
+      const migrationArticles = await articles.find().toArray()
+
+      for (const article of migrationArticles) {
+        if (article.draft) {
```

Comment:
Why not using the mongo internal function  "$toString" insted of iterating over every article?

https://www.mongodb.com/docs/manual/reference/operator/aggregation/toString/

Reply:
@tomaszdurka: @penk85  told me you told her to do it the "unsave" way. Don't you think we should use atomic operations for data manipulation that there is less risk of failing operation?

Reply:
Or maybe use Transactions: https://www.digitalocean.com/community/tutorials/how-to-use-transactions-in-mongodb

Reply:
@elias-summermatter transactions for migrations will be introduced in a different pull request. If that's okay, can you approve?

Reply:
Unfortunately transactions in Mongodb require a replica set, we currently are only running a standalone server.